### ISSUE TITLE
Unable to build UE docker image

### ISSUE THREAD

**Issue by papa-singh** [adamrehn/ue4-docker#301]
Output of the `ue4-docker info` command:

```
ue4-docker version:         0.0.103 (latest available version is 0.0.103)
Operating system:           Linux (Ubuntu 20.04.5 LTS, 5.15.0-67-generic)
Docker daemon version:      23.0.1
NVIDIA Docker supported:    No
Maximum image size:         No limit detected
Available disk space:       332.47 GiB
Total system memory:        125.43 GiB physical, 30.52 GiB virtual
CPU:                        12 physical, 12 logical (x86_64)

```

Additional details:

- Are you accessing the network through a proxy server? **No**

I'm trying to build UE 4.20.3 using the command `ue4-docker build 4.20.3`. I ran `ue4-docker setup` first and it returned "ue4-docker build 4.20.3". However the `build` command returns the following error:

```
Traceback (most recent call last):
  File "/home/ram/.local/bin/ue4-docker", line 8, in <module>
    sys.exit(main())
  File "/home/ram/.local/lib/python3.8/site-packages/ue4docker/main.py", line 109, in main
    COMMANDS[command]["function"]()
  File "/home/ram/.local/lib/python3.8/site-packages/ue4docker/build.py", line 52, in build
    config = BuildConfiguration(parser, sys.argv[1:], logger)
  File "/home/ram/.local/lib/python3.8/site-packages/ue4docker/infrastructure/BuildConfiguration.py", line 519, in __init__
    if self.opts["credential_mode"] not in validCredentialModes:
TypeError: string indices must be integers
```

How can I solve this issue?

**Comment by TBBle** [comment]
Looks like a small bug at https://github.com/adamrehn/ue4-docker/pull/295/files#diff-2faa50ca4c6d80d7dd0e093c700a90935e359adf3b9ea640a108c2b8d1cbd9edR509-R511

As a workaround, add `--credential-mode=secrets` to achieve what that change was supposed to do.

Tagging @slonopotamus since the fix looks easy (`self.opts["credential_mode"] = (`, but I'm not in a position to actually fix it.

**Comment by slonopotamus** [comment]
Whoops. Not sure how this has happened.

**Comment by slonopotamus** [comment]
I also managed to [break publishing to PyPI](https://github.com/adamrehn/ue4-docker/actions/runs/4384266053/jobs/7675532447). Whoopsie-doopsie. Fixing...

**Comment by slonopotamus** [comment]
Published 0.0.105 release with a fix for this issue.
